README.md

```diff
@@ -8,7 +8,7 @@ Estimating the geographical range of a species from sparse observations is a cha
 ![Model Prediction](images/sinr_traverse.gif)
```

Comment:
I think you need to pull from main, cause there's a title that makes it clear it's our forked repository